wdio.conf.js

```diff
@@ -0,0 +1,167 @@
+const uuid = require('uuid/v4');
+const browserstack = require('browserstack-local');
+
+// Stop node from complaining about fake memory leaks at higher concurrency
+require('events').defaultMaxListeners = 20;
+require('dotenv').config();
+
+const ONLY_BROWSERS = process.env.ONLY_BROWSERS;
+const localIdentifier = uuid();
+const screenResolution = '1920x1080';
+
+const defaultCapabilities = {
+  'browserstack.debug': true,
+  'browserstack.local': true,
+  'browserstack.networkLogs': true,
+  'browserstack.localIdentifier': localIdentifier
+};
+
+const desktopCapabilities = {
+  ...defaultCapabilities,
+  os: 'windows',
+  os_version: '10',
+  resolution: screenResolution
+};
+
+const mobileCapabilities = {
+  ...defaultCapabilities,
+  real_mobile: true,
+  'browserstack.appium_version': '1.14.0'
+};
+
+let capabilities = [
+  {
+    ...desktopCapabilities,
+    browserName: 'Google Chrome',
+    browser: 'chrome',
+    'browserstack.console': 'info'
+  },
+  // TODO mobile browsers are having trouble with browser.keys
+  //{
+  //  ...mobileCapabilities,
+  //  browserName: 'iPhone XS Safari',
+  //  device: 'iPhone XS',
+  //  os_version: '13'
+  //},
+  //{
+  //  ...mobileCapabilities,
+  //  browserName: 'Google Pixel 3 Chrome',
+  //  device: 'Google Pixel 3',
+  //  os_version: '9.0',
+  //  chromeOptions: {
+  //    prefs: {
+  //      // disable chrome's annoying password manager, may be unnec
+  //      'profile.password_manager_enabled': false,
+  //      credentials_enable_service: false,
+  //      password_manager_enabled: false
+  //    }
+  //  }
+  //},
+  {
+    ...desktopCapabilities,
+    browser: 'IE',
+    browserName: 'IE 11',
+    browser_version: '11.0',
+    'browserstack.selenium_version' : '3.5.2'
+  },
+  {
+    ...desktopCapabilities,
+    browserName: 'Microsoft Edge',
+    browser: 'edge',
+    browser_version: '18.0'
+  },
+  {
+    ...desktopCapabilities,
+    browserName: 'Firefox',
+    browser: 'firefox',
+    'browserstack.console': 'info'
+  },
+  {
+    ...desktopCapabilities,
+    browserName: 'Desktop Safari',
+    browser: 'safari',
+    os: 'OS X',
+    os_version: 'Mojave'
+  }
+];
+
+if (ONLY_BROWSERS) {
+  capabilities = ONLY_BROWSERS.split(',').map(browser => capabilities.find(config => config.browser.toLowerCase() === browser.toLowerCase()))
```

Comment:
This line is missing a semicolon but also it's super long and rather difficult to parse. How does breaking it up like this work for you?
```suggestion
  capabilities = ONLY_BROWSERS
    .split(',')
    .map(browser => capabilities
      .find(config => config.browser.toLowerCase() === browser.toLowerCase())
    );
```